warp10/src/main/java/io/warp10/script/binary/DIV.java

```diff
@@ -55,6 +55,14 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
       GeoTimeSerie gts1 = (GeoTimeSerie) op1;
       GeoTimeSerie gts2 = (GeoTimeSerie) op2;
 
+      // Returns immediately a new gts if both inputs are empty
+      if (0 == GTSHelper.nvalues(gts1) || 0 == GTSHelper.nvalues(gts2)) {
+        GeoTimeSerie result = new GeoTimeSerie();
+        result.setType(TYPE.DOUBLE);
```

Comment:
Until now I think all empty GTSs are of type UNDEFINED. I understand the logic but are you sure it does not break anything?
What is your opinion @hbs?